### PR TITLE
Add github actions for running tests

### LINKED ISSUE #13: Setup GH Actions

### PR DESCRIPTION
**Pull request by cutz**
Fixes #13﻿

### CODE REVIEW

**Comment by jamadden** [comment]
It’s good to remove the .travis.yml too

**Comment by cutz** [comment]
Well, making a bit of progress. Tests are failing due to the strict IRO checking. I'm not 100% sure how to interpret the output here. Are these issues in zope.publisher land?

Issue #16

**Comment by cutz** [comment]
Not sure why coveralls is reporting no coverage now.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip setuptools wheel
+        python -m pip install -U coverage
+        python -m pip install -U -e ".[test,docs]"
+    - name: Test
+      run: |
+        coverage run -m zope.testrunner --test-path=src  --auto-color --auto-progress
+        coverage run -a -m sphinx -b doctest -d docs/_build/doctrees docs docs/_build/doctests
```
Comment (jamadden) [inline .github/workflows/tests.yml:37]: In more recent workflows, I add a step after this that does 'coverage report' so we can see what's happening there.

**Review by jamadden — COMMENTED** [review]
> Not sure why coveralls is reporting no coverage now.

Probably because there's a setting that needs added in .coveragerc, `relative_files = true`. See nti.webhooks for an example and comment.

**Review by jamadden — APPROVED** [review]

**Pull request merged**